tools/e2e/commands.ts

```diff
@@ -152,6 +153,15 @@ Cypress.Commands.add('clearRecordDrafts', () => {
   cy.reload()
 })
 
+Cypress.Commands.add('publishAndReload', () => {
+  cy.wait(1200)
```

Comment:
What do we need this first wait for? We really try to wait as little as possible. Can it be replaced by checking for a specific state?

Reply:
It waits for the form to "realize" that a change has been made, quite similar to waiting for the draft to be saved in the local storage. Without that, the save command doesn't work.